druid/src/main/java/org/apache/calcite/adapter/druid/DruidRules.java

```diff
@@ -140,18 +140,28 @@ public boolean test(Triple<Aggregate, RelNode, DruidQuery> triple) {
           for (AggregateCall aggregateCall : aggregate.getAggCallList()) {
             switch (aggregateCall.getAggregation().getKind()) {
             case COUNT:
-              // Druid can handle 2 scenarios:
+              // Druid count aggregator can handle 3 scenarios:
               // 1. count(distinct col) when approximate results
               //    are acceptable and col is not a metric
+              // FYI case exact count(distinct column) is handled using group by and count as well
```

Comment:
Is there a follow-up JIRA for this? If there is not, can we create it? Then we can reference it here.

Reply:
no am saying that we are handling this case already and the planer already does write it as druid group by.

Reply:
OK, got it, thanks